bin/test

```diff
@@ -3,9 +3,9 @@
 set -e -x
 
 go vet ./...
-go install -v github.com/onsi/ginkgo/ginkgo
+go install -v github.com/onsi/ginkgo/v2/ginkgo
 packages=("http_routes" "tcp_routing" "smoke_tests")
 for i in "${packages[@]}"
 do
-  ginkgo -r -race -slowSpecThreshold=120 "$@" "$i"
```

Comment:
Any reason we didn't [replace](https://github.com/cloudfoundry/garden/commit/76fb7bb00c075806f68546c39d933289c56773f4) this with `--slow-spec-threshold`?

Reply:
We decided that we never used the output of the slowSpec reporting anyway, or look at it in the context of CI. If we want to explicitly look into fixing test times, we can add it via the $@ parameter, and keep the test output cleaner for the majority of cases.

Reply:
Additionally, the `--slow-spec-threshold` flag is *also* deprecated now, in favor of another reporting mechanism.